README.md

```
@@ -12,20 +11,13 @@
   <h3>FileBrowser Quantum</h3>
   The best free self-hosted web-based file manager.
   <br/><br/>
-  <img width="800" src="https://github.com/user-attachments/assets/162d7a95-33b7-49bd-976c-dd6822c0d22b">
+  <img width="800" src="https://filebrowserquantum.com/images/generated/listing/view-mode-normal-dark.jpg">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)